File: src/targets/php/guzzle/fixtures/nested.php
<?php
<<<<<<< HEAD
require_once('vendor/autoload.php');

$client = new \GuzzleHttp\Client();

$response = $client->request('GET', 'https://httpbin.org/anything?foo%5Bbar%5D=baz%2Czap&fiz=buz&key=value');
=======

$client = new \GuzzleHttp\Client();

$response = $client->request('GET', 'http://mockbin.com/har?foo%5Bbar%5D=baz%2Czap&fiz=buz&key=value');
>>>>>>> upstream/master

echo $response->getBody();

File: src/targets/csharp/restsharp/fixtures/nested.cs
<<<<<<< HEAD
var client = new RestClient("https://httpbin.org/anything?foo%5Bbar%5D=baz%2Czap&fiz=buz&key=value");
=======
var client = new RestClient("http://mockbin.com/har?foo%5Bbar%5D=baz%2Czap&fiz=buz&key=value");
>>>>>>> upstream/master
var request = new RestRequest(Method.GET);
IRestResponse response = client.Execute(request);